docs/containerd-env-setup.md

```diff
@@ -56,28 +56,37 @@ $ sudo tee /etc/nydusd-config.json > /dev/null << EOF
 EOF
 ```
 
-Note:
+Please refer to the nydusd [doc](./nydusd.md) to learn more options.
+
+⚠️ Note:
+
+- The `device.backend.config.scheme` is registry url scheme, leave it empty to automatically detect, otherwise specify to `https` or `http` according to your registry server configuration.
+- The `device.backend.config.auth` is base64 encoded `username:password`, it is required by `nydusd` to lazily pull image data from registry which is authentication enabled, the nydus snapshotter will automatically read docker login auth from the configuration `$HOME/.docker/config.json`, otherwise please copy it to replace `YOUR_LOGIN_AUTH=`.
```

Comment:
- The device.backend.config.auth is the base64 encoded username:password required by nydusd to lazily pull image data from an authenticated registry. The nydus snapshotter will automatically read the authentication information from the $HOME/.docker/config.json configuration file. If you are using a registry that requires authentication, you should replace YOUR_LOGIN_AUTH= with your own login information.